docs/tutorials/using_apis/mesh/quick_start_build.md

```diff
@@ -1,6 +1,6 @@
-### Build time configuration of the stack
+### Build time configuration of the Nanostack
 
-To minimize the size of the produced stack, Nanostack defines a set of build options.
+The build time configuration of Nanostack uses the Mbed configuration system. The application needs to create an `mbed_app.json` configuration file if you want to use settings other than default settings. You can also minimize the size of the produced stack by using different build options.
 
 #### Build options
```

Comment:
The below table shows binary size in Mbed OS 5.5. Because we're going into 5.8, should we update the table, or is it OK as is?

Reply:
Could we just mention something like -> 
The binary sizes in the table are just for showing the difference between different builds.